### PR TITLE
Add classification for Microsoft.VisualBasic.dll

### PR DESCRIPTION
**Pull request by vatsan-madhavan**
- Add classification for M.VB.dll
- Ignore M.VB.Forms.dll during ref-checks.

/cc @dagood @AdamYoblick @rladuca

### CODE REVIEW

**Comment by vatsan-madhavan** [comment]
My claim about M.VB.Forms not having a ref assembly is wrong. 

@dagood pointed me to https://github.com/dotnet/winforms/pull/2396/files. 
Also the latest winforms transport package has ref assembly for m.winforms.vb.dll.

**Comment by vatsan-madhavan** [comment]
Let's start by adding classification for M.Vb.dll and see what happens to the build...

File: pkg/windowsdesktop/pkg/Directory.Build.props [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
     <FrameworkListFileClass Include="Accessibility.dll" Profile="WindowsForms;WPF" />
+    <FrameworkListFileClass Include="Microsoft.VisualBasic.dll" Profile="WindowsForms" />
```
Comment (RussKie) [inline pkg/windowsdesktop/pkg/Directory.Build.props]: Do we need to add `Microsoft.VisualBasic.Forms` here?

Comment (vatsan-madhavan) [reply pkg/windowsdesktop/pkg/Directory.Build.props]: I tried doing that I'm getting errors. I'm going to add that in a separate step to see what happens next.

**Pull request merged**